pkg/volume/downwardapi/downwardapi.go

```diff
@@ -262,6 +261,29 @@ func CollectData(items []v1.DownwardAPIVolumeFile, pod *v1.Pod, host volume.Volu
 	return data, utilerrors.NewAggregate(errlist)
 }
 
+// TODO: support "status.podIP" once we can get pod ip before first setup volume.
```

Comment:
Are there any use cases for status.podIP? I think containers could get its IP easily with any networking tools or library if required.

Reply:
yes, I can remove this TODO

Reply:
We support podIP on env vars, and we do it explicitly because networking tools and libraries are *terrible* at telling callers what the "correct IP" is.  Having it in the downward API makes the behavior predictable and consistent with how the pod will be exposed by services.

I assume, looking at the PR, that you didn't fix the behavior.  The TODO should remain until it's fixed.

Reply:
OK